examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -49,10 +50,10 @@
   private long mId;
 
   public BasicRawTableOperations(TachyonURI masterAddress, TachyonURI tablePath,
-      WriteType writeType) {
+      TachyonStorageType writeType) {
     mMasterAddress = masterAddress;
     mTablePath = tablePath;
-    mWriteType = writeType;
+    mWriteType = writeType.isStore() ? WriteType.MUST_CACHE : WriteType.THROUGH;
```

Comment:
Ah, ok. Then, should the outer command line take both TachyonStorageType and UnderStorageType, and all combinations are combined to translate to WriteType?

The reason I am asking is that CACHE_THROUGH is a typical configuration, which would be impossible going forward.